main.tf

```diff
@@ -13,6 +13,7 @@ data "ibm_iam_account_settings" "iam_account_settings" {
 module "cbr_zone" {
   source             = "./cbr-zone-module"
   name               = var.name
+  region             = var.region
```

Comment:
It looks like there is some code missing in this PR as the ./cbr-zone-module is not updated here to take the region as input.

Reply:
<img width="796" alt="Screenshot 2023-06-20 at 2 18 09 AM" src="https://github.com/terraform-ibm-modules/terraform-ibm-cbr/assets/122350533/885e2331-cc48-4dc0-9507-ad79de68fadb">


Region does not seems to be an expected argument in the "cbr_zone" module